django/utils/cache.py

```diff
@@ -92,12 +92,11 @@ def get_max_age(response):
     """
     if not response.has_header('Cache-Control'):
         return
-    cc = dict(_to_tuple(el) for el in cc_delim_re.split(response['Cache-Control']))
-    if 'max-age' in cc:
-        try:
-            return int(cc['max-age'])
-        except (ValueError, TypeError):
-            pass
+    cc = dict(map(_to_tuple, cc_delim_re.split(response['Cache-Control'])))
```

Comment:
Usually we've preferred list comprehensions as more readable than map.

Reply:
OK, if you think it is not readable enough even in this case where the conversion function is already defined, I can change this back. I generally prefer functional style as long as it does not hurt readability, and this is the case when I thought it doesn't.